etcdmain/config.go

```diff
@@ -210,6 +210,7 @@ func newConfig() *config {
 	fs.StringVar(&cfg.ec.PeerTLSInfo.CRLFile, "peer-crl-file", "", "Path to the peer certificate revocation list file.")
 	fs.StringVar(&cfg.ec.PeerTLSInfo.AllowedCN, "peer-cert-allowed-cn", "", "Allowed CN for inter peer authentication.")
 	fs.Var(flags.NewStringsValue(""), "cipher-suites", "Comma-separated list of supported TLS cipher suites between client/server and peers (empty will be auto-populated by Go).")
+	fs.BoolVar(&cfg.ec.PeerTLSInfo.SkipClientVerify, "peer-skip-client-verify", false, "Skip client IP verification for peer connections.")
```

Comment:
`Skip client SAN field verification for peer connections`?

Reply:
I think the skip client verify is a very vague description. can we make it clear to include what exactly is skipped?

Reply:
Changed description to `Skip verification of SAN field in client certificate for peer connections.`
Any more ideas?

Reply:
> Skip verification of SAN field in client certificate for peer connections

This is clearer.